crates/project/src/dap_store.rs

```diff
@@ -145,8 +145,24 @@ impl DapStore {
     pub fn new_remote(
         project_id: u64,
         upstream_client: AnyProtoClient,
+        breakpoints: Option<Vec<proto::SynchronizeBreakpoints>>,
         _: &mut ModelContext<Self>,
     ) -> Self {
+        let breakpoints = match breakpoints {
```

Comment:
I'm not sure if I like this approach. I feel like we have to add this somewhere else, this feels like a hack,